docs/sql-data-sources-jdbc.md

```diff
@@ -213,6 +213,14 @@ the following case-insensitive options:
   </tr>
 </table>
 
+It worth to mention that kerberos authentication with keytab is not always supported by the JDBC driver.<br>
+Before using <code>keytab</code> and <code>principal</code> configuration options please make sure the following requirements met:
+<ol>
+   <li> The included JDBC driver version supports kerberos authentication with keytab. </li>
+   <li> There is an embedded connection provider which supports the used database (many database support already added, but since it's changing please double check the list). </li> 
+</ol>
+If any other kind of custom authentication is needed <code>JdbcConnectionProvider</code> developer API is added which allows developers to implement it.
```

Comment:
How about `If the requirements are not met, please consider using the <code>JdbcConnectionProvider</code> developer API to handle custom authentication.`?

Reply:
Changed.